module/zfs/arc.c

```diff
@@ -7750,6 +7883,9 @@ arc_init(void)
 
 	arc_prune_taskq = taskq_create("arc_prune", zfs_arc_prune_task_threads,
 	    defclsyspri, 100, INT_MAX, TASKQ_PREPOPULATE | TASKQ_DYNAMIC);
+	if (likely(max_ncpus > 1))
+		arc_evict_taskq = taskq_create("arc_evict", max_ncpus,
+		    defclsyspri, 0, INT_MAX, TASKQ_PREPOPULATE);
```

Comment:
So now my system will get 128 kernel threads just for fun, that it will never use, only make the parameter configurable?